zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -92,11 +99,12 @@ object Server {
   def start[R <: Has[_]](
     port: Int,
     http: HttpApp[R, Throwable],
-  ): ZIO[R, Throwable, Nothing] =
+  ): ZIO[R, Throwable, Nothing] = {
```

Comment:
```suggestion
  ): ZIO[R, Throwable, Start] = {
```

Reply:
@tusharmath it's already supported in a way in the make method.